maintainers/maintainer-list.nix

```diff
@@ -736,6 +736,12 @@
     githubId = 1771266;
     name = "Vo Anh Duy";
   };
+  anirrudh = {
```

Comment:
can you it a separate commit: `maintainers: add anirrudh` that goes before your current commit?

Reply:
Yep! I apologize - I squashed everything into one!